.github/CODEOWNERS

```diff
@@ -626,8 +632,9 @@
 
 
 # User Data Takeout.
-/core/domain/takeout_*.py @BenHenning @vojtechjelinek
-/core/domain/wipeout_*.py @BenHenning @vojtechjelinek
+# TODO(#14285): Re-add @BenHenning for both.
```

Comment:
@BenHenning Please replace you with me instead, here (I would like to at least be notified for these PRs).

Reply:
Done.